app/src/ui/mapview/Mapview.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import clsx from "clsx";
+import styles from "./Mapview.module.scss";
+
+import { MapViewProps } from "./Mapview.types";
+import { MapViewProvider } from "hooks/useMapContext/useMapContext";
+
+export const MapView: React.FC<MapViewProps> = ({ children, className, mapOptions }) => {
+  const mapRootElementRef = React.useRef<HTMLDivElement>(null);
+  const [currentMap, setCurrentMap] = React.useState<google.maps.Map>();
+
+  React.useEffect(() => {
+    if (mapRootElementRef.current && !currentMap) {
+      const newMap = new window.google.maps.Map(mapRootElementRef.current, { ...mapOptions });
+      setCurrentMap(newMap);
+    }
+  }, [mapRootElementRef, currentMap, mapOptions]);
+
+  return (
+    <div ref={mapRootElementRef} className={clsx(styles["map-view"], className)}>
+      <MapViewProvider
+        value={{
+          map: currentMap,
+          mapOptions,
+        }}
+      >
+        {children}
+      </MapViewProvider>
```

Comment:
This goes inside a `MapContextController`: 

https://github.com/aufacicenta/bancosatoshi/blob/master/app/src/context/tab/TabContextController.tsx


Normally, `Context.Provider`s go inside an `AppLayout`. In this case, you'll need to put it in the "sala de inversión" page: 

```
const Map: NextPage = () => (
  <AppLayout>
    <AuthLayout>
      <MapContextController>
          <Map.Container>
             <Map />
          </Map.Container>
      </MapContextController>
    </AuthLayout>
  </AppLayout>
);
```